content/applications/sales/sales/products_prices/returns.rst

```diff
@@ -1,73 +1,114 @@
+
+On the validated delivery order, click :guilabel:`Return`.
+
+.. image:: returns/validated-delivery-order-return-button.png
+   :align: center
+   :alt: A validated delivery order with a highlighted Return button in Odoo Sales.
+
+This opens a :guilabel:`Reverse Transfer` pop-up window.
+
+By default, the :guilabel:`Quantity` matches the validated quantities from the delivery order.
+Update the quantities, if necessary. Click on the :guilabel:`🗑️ (trash)` icon next to a line item to
 remove it from the return.
 
 .. image:: returns/reverse-transfer-popup.png
    :align: center
    :alt: The "Reverse Transfer" pop-up window, to make a return before invoicing the customer.
 
 Next, click :guilabel:`Return` to confirm the return. This generates a new warehouse operation for
-the incoming returned product(s). Upon receiving the return, the warehouse team validates the
-warehouse operation. Then, on the original sales order, the :guilabel:`Delivered` quantity will
-reflect the difference between the initial validated quantities and the returned quantities.
+the incoming returned product(s).
+
+.. image:: returns/warehouse-operation-confirmed-return.png
+   :align: center
+   :alt: Warehouse operation after a return has been confirmed in Odoo Sales.
+
+Upon receiving the return, the warehouse team validates the warehouse operation. Then, on the
+original sales order, the :guilabel:`Delivered` quantity updates to reflect the difference between
+the initial validated quantities and the returned quantities.
 
 .. image:: returns/updated-sales-quantities.png
    :align: center
    :alt: The updated "Delivered" quantity on the sales order after the reverse transfer.
 
-When an invoice is created, the customer receives an invoice only for the products they are
+When an invoice is created, the customer receives an invoice **only** for the products they are
 keeping.
 
 After invoicing
 ===============
 
 Sometimes, customers return an item after they receive and/or pay for their invoice. In these
-cases, a return using only *Reverse Transfers* is insufficient since validated or sent invoices
-cannot be changed. However, *Reverse Transfers* can be used in conjunction with *Credit Notes* to
-complete the customer's return.
+cases, a return using only *Reverse Transfers* is insufficient since validated, or sent, invoices
+cannot be changed.
+
+However, *Reverse Transfers* can be used in conjunction with *Credit Notes* to complete the
+customer's return.
+
+To start a return after invoicing, navigate to the relevant sales order in the *Sales* app.
 
-To start a return, navigate to the relevant sales order. If there is a payment registered on the
-sales order, then the payment details will appear in the Chatter, and the invoice (accessible
-through the :guilabel:`Invoices` smart button) will have a green banner across it.
+If there is a payment registered on the sales order, the payment details appear in the Chatter, and
```

Comment:
I'm not entirely sure what formatting is best to use when mentioning the Chatter window, but it might need a GUI label.

Reply:
Yeah -- good point. I think that, since the word "Chatter" isn't directly on the U/I, and the section itself is merely referred to as "Chatter" -- I think I'll leave it without a GUI label. But, if this is incorrect, I'll be sure to go back and make the necessary adjustment. Thanks for pointing this out. 👍